web/modules/custom/asu_migrate/src/Plugin/Field/FieldFormatter/ComplexTitleCSVFormatter.php

```diff
@@ -27,9 +27,9 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       $nonsort = $item->entity->field_nonsort->value . '';
       $main = $item->entity->field_main_title->value;
       $sub = $item->entity->field_subtitle->value . '';
-      $nm = $nonsort . ":" .
-        ($main != NULL ? $main : "[untitled]") . ":" .
-        $sub;
+      $nm = (($nonsort) ? $nonsort . ':' : '').
```

Comment:
Is this putting a `:` between the nonsort and the main part of the title?

Reply:
yes but only if there is a value for nonsort. I might need to go back and redo this, but in my round-trip, the import worked fine -- though I didn't give my test item any value for the nonsort. Did that do something weird in your testing?

Reply:
there just shouldn't be anything but a space between nonsort and the main part of the title.
ie if the nonsort is "The" and the main part of the title is "Awesome" then the combined title should be "The Awesome" not "The: Awesome"

Reply:
oh crap! That has to change then. Lemme just edit the code via git unless you have already.

Reply:
i have not edited it. go for it